ConsoleApp/Program.cs

```diff
@@ -0,0 +1,132 @@
+﻿internal class Program
+{
+    static int MaxBetween (int a, int b)
+    {
+        if (a>b)
+            return a;
+        else 
+            return b;
+    }
+
+    static int MaxBetween (int a, int b, int c)
+    {
+        int[] numbers = new int[] {a, b, c};
```

Comment:
please rewrite it without using arrays (the same to all methods where you've used them)